web/tests/Web.E2ETests/Extensions/LocatorExtensions.cs

```diff
@@ -151,4 +151,16 @@ public static async Task<ILocator> AssertLocatorClass(this ILocator locator, str
         await Assertions.Expect(locator).ToHaveClassAsync(locatorClass);
         return locator;
     }
+
+    public static async Task<ILocator> PressSequentially(this ILocator locator, string inputValue)
+    {
+        await locator.PressSequentiallyAsync(inputValue, new() { Delay = 100 });
```

Comment:
I assume we need to handle keyboard press which is why this is being used rather than FillAsync?

>In most cases, you should use FillAsync(string, LocatorFillOptions?) instead. You only need to press keys one by one if there is special keyboard handling on the page.

Reply:
yes that is correct. using fillAsync is not showing the suggester dropdown hence the other one is used.